src/openshift/component.ts

```diff
@@ -203,6 +205,63 @@ export class Component extends OpenShiftItem {
         return Component.dev(component, 'podman');
     }
 
+    @vsCommand('openshift.component.binding.add')
+    static async addBinding(component: ComponentWorkspaceFolder) {
+        const odo: odo3.Odo3 = odo3.newInstance();
+
+        const services = await odo.getBindableServices();
+        if (!services || services.length === 0) {
+            throw new Error('No bindable services are available');
```

Comment:
Could we direct the user here in the message to somewhere where they can add a service?

Reply:
I added a button that links to the operator backed service catalog.